src/ast/value.rs

```diff
@@ -59,6 +59,8 @@ pub enum Value {
     },
     /// `NULL` value
     Null,
+    /// `?` or `$` Prepared statement arg placeholder
```

Comment:
I am not familiar with the `$` syntax for prepared argument placeholders. What database system do they come from?

I have only ever seen `?` used for prepared statement placeholders

Reply:
Here is an [example](https://www.postgresql.org/docs/current/sql-prepare.html#:~:text=pg_prepared_statements%20system%20view.-,Examples,-Create%20a%20prepared) from pgsql's document.
```sql
PREPARE fooplan (int, text, bool, numeric) AS
    INSERT INTO foo VALUES($1, $2, $3, $4);
EXECUTE fooplan(1, 'Hunter Valley', 't', 200.00);
```
However, only numbers follow the $.
Then I found that there was a need for $foo in the issue. https://github.com/sqlparser-rs/sqlparser-rs/issues/145#issuecomment-557332621

So the final product is this:
```sql
SELECT * FROM student WHERE id = $Id1;
```
Maybe there can't be a separate $placeholder here.

Thank you even more for your continued contribution. I'm just a spoon of water in the ocean :-)